e2e/rbd.go

```diff
@@ -3383,6 +3385,114 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("restore snapshot to a bigger size PVC", func() {
```

Comment:
what is the benefit of these two nested `By()` calls?

Reply:
Having nested `by` helps to break larger testing into smaller pieces for better readability.